README.md

```diff
@@ -1 +1,16 @@
-# MediaMonks - Frontend Coding Standards
+# frontend-coding-standards
+
+## GIT
+
+### Branches
+We use [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) for our branching strategy.
+
+#### Automatic deployment of branches
+Some projects will automatically deploy to an environment when pushing commits into a specific branch. Which branch is connected to which environment should be written in the `README.md` of the project.
+
+### Commit messages
+Please read: https://chris.beams.io/posts/git-commit/
```

Comment:
https://www.conventionalcommits.org/en/v1.0.0-beta.2/ ?

Reply:
I created an issue for this: https://github.com/mediamonks/frontend-coding-standards/issues/20